javagony/OddOrEvenChecker.java

```diff
@@ -0,0 +1,15 @@
+import java.util.Scanner;
+
+public class OddOrEvenChecker {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        int number = scanner.nextInt();
+        try{
+            int remainder = 1/(number%2);
+            System.out.print(number+" is odd");
```

Comment:
```suggestion
            System.out.printf("%d is odd\n", number);
```
This is more of a stylistic choice, but I think that `.printf` is more readable in this case :slightly_smiling_face:

Reply:
I'll make the necessary changes and update it